env/index.js

```diff
@@ -51,15 +51,19 @@ function setEnvironmentVarsFromTestEnvYaml(name) {
     return;
   }
 
+  if (doc[name]) {
+    console.log(`Loading environment variables from testenv.yml: "${name}"`);
```

Comment:
It was intended to load from "default" for every section, then allow the named section to add overrides, or additional variables.  With this change it will load from named section OR from default, requiring the section to contain ALL variables.

Reply:
reverted